doc/source/reference/arrays.scalars.rst

```diff
@@ -248,7 +248,7 @@ Indexing
 Array scalars can be indexed like 0-dimensional arrays: if *x* is an
 array scalar,
 
-- ``x[()]`` returns a 0-dimensional :class:`ndarray`
+- ``x[...]`` returns a 0-dimensional :class:`ndarray`
```

Comment:
I see that there is a little difference between 0-d arrays and scalars from users point of view, but:
```python
>>> a = np.int_(1)
>>> type(a)
numpy.int32
>>> type(a[...])
numpy.ndarray
```
on my computer.